docs/pages/installation/eks.md

```diff
@@ -6,7 +6,18 @@ permalink: installation/eks
 
 # Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
-## Managed Kubernetes
+Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu. Utilize the token to run the following command:
+
+| command           | flag                | function                                                     | Usage                     |
+|:------------------|:-------------------:|:-------------------------------------------------------------|:--------------------------|
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(EKS). | `mesheryctl system config gke --token "PATH TO TOKEN"` |
+
```

Comment:
I wonder if we would do well to include a final step of "Once configured, continue with the steps outlined in the Quick Start Guide (hyperlink) for your respective platform."

Reply:
Let's go ahead and include this final instruction.

Reply:
Inclusion of the table is out of place here. Keep it simple and just give the user the command they should run.